docs/sidebars.js

```diff
@@ -25,6 +25,23 @@ module.exports = {
       label: "Quickstart",
       id: "quickstart",
     },
+    {
+      type: "category",
+      label: "Getting Started",
+      link: {
+        type: "generated-index",
+        title: "Getting Started",
+        description:
+          "This section describes how to install and deploy Nebari on a selection of cloud providers, and also provides details on how Nebari can be configured and customized to fit a variety of deployment requirements.",
+        slug: "category/started",
+        keywords: ["started"],
+      },
+      items: [
+        "started/installing-nebari",
```

Comment:
And since this is the "start here: directory this makes the quickstart somewhat redundant

Reply:
ah. I didn't see this comment. Yes, I agree. 

The quickstart was/is not a directory, just a single file. It should live with the getting started docs. 

I agree with renaming the directory for these docs from `started` to `getting-started`

Reply:
I was a little confused earlier on the quickstart. It shouldn't be a file, it should be a directory. But since we now have `getting-started`, the `quickstart` file should be removed.